sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -1649,4 +1649,17 @@ class JDBCSuite extends QueryTest
       }
     }
   }
+
+  test("SPARK-29675 Add exception when isolationLevel is Illegal") {
```

Comment:
Super nit. @ulysses-you . We don't use JIRA ID for the improvement test cases. It's used for bug fixes only.

Reply:
I discussed with @dongjoon-hyun offline. I will send an email to dev mailing list to discuss and officially document this test prefix for JIRA ID, after having some more offline discussions with some other PMCs.

Reply:
Looks nice, thx, PMC guys!